tachybradycardia.py

```diff
@@ -0,0 +1,30 @@
+def bradycardia(hrinst, lowerthresh):
```

Comment:
It might be good to roll this in as a more native method that can utilize the object attributes, rather than having to pass in variables.  (One of the benefits of using the OOP approach.)